lib/test-endpoint.sh

```diff
@@ -7,22 +7,39 @@ DD_US_API_SITE="https://api.datadoghq.com/api/"
 DD_API_SITE=$DD_US_API_SITE
 DD_USE_EU=false
 
+DD_DEFAULT_HTTPS_EU_ENDPOINT="agent-http-intake.logs.datadoghq.eu:443"
+DD_DEFAULT_HTTPS_US_ENDPOINT="agent-http-intake.logs.datadoghq.com:10516"
+DD_DEFAULT_TCP_EU_ENDPOINT"agent-intake.logs.datadoghq.eu:443"
+DD_DEFAULT_TCP_US_ENDPOINT="agent-intake.logs.datadoghq.com:10516"
+
 if [ -n "$DD_SITE" ] && [ "$DD_SITE" = "datadoghq.eu" ]; then
   DD_USE_EU=true
   DD_API_SITE=$DD_EU_API_SITE
 fi
 
+if [ -n "$DD_LOGS_CONFIG_USE_HTTP" ] && [ "$DD_LOGS_CONFIG_USE_HTTP" = true]; then
```

Comment:
no need for the first condition
```suggestion
if [ "$DD_LOGS_CONFIG_USE_HTTP" = true]; then
```